README.md

```diff
@@ -56,6 +56,51 @@ We provide the following ways for you to use xrefcheck:
 
 If none of those are suitable for you, please open an issue!
 
+If you want some external links to not be verified, you can use one of the following ways to ignore that links:
+1. Add the regular expression that match ignoring link to optional `ignoreRefs` parameter of your config file.
+
+For example: 
+```sh
+ignoreRefs:
+  - https://bad.reference.(ru|com)(/?)
+```
+allows to ignore both `https://bad.reference.ru` and `https://bad.reference.com` with or without last "/".
+
+2. Add right in-place annotation using one of the following ignoring mode (each mode is just a comment with a certain syntax).
+
+* Ignore the link:
+
+```sh
+<!-- xrefcheck: ignore link -->
+[Bad 'ru' reference](https://bad.reference.ru)
+[Bad 'com' reference](https://bad.reference.com)
+```
+
+Therefore only `https://bad.reference.ru` will be ignored.
+
+* Ignore the paragraph:
+
+```sh
+<!-- xrefcheck: ignore paragraph -->
+[Bad 'ru' reference](https://bad.reference.ru)
+[Bad 'com' reference](https://bad.reference.com)
+
+[Bad 'io' reference](https://bad.reference.io)
+```
```

Comment:
I think it's also wroth including some text, not bare links, otherwise the reader may think that ignorance works only on standalone links.